tests/acceptance/features/apiLocks/lockFiles.feature

```diff
@@ -312,7 +312,6 @@ Feature: lock files
       | new              |
       | spaces           |
 
-  @issue-7638
```

Comment:
@2403905 Let's not remove the issue tags. it servers as a reference for the future.

Reply:
And if the tags do get removed, then leave a blank line (2 lines between scenarios)
If the line numbers of scenarios change, then it is a nuisance for the expected-failures files.